src/stan/services/init/initialize_state.hpp

```diff
@@ -24,7 +25,7 @@
 namespace stan {
   namespace services {
     namespace init {
-
+      using stan::interface_callbacks::writer::base_writer;
```

Comment:
Sorry, meant the using.  That carries over to the entire namespace, right?
Which means that another file in init can use “base_writer” without a scope
and there won’t be any local indication of what that scope might be.

On Nov 7, 2015, at 5:06 PM, Daniel Lee <notifications@github.com> wrote:

> In src/stan/services/init/initialize_state.hpp:
> 
> > @@ -24,7 +25,7 @@
> >  namespace stan {
> >    namespace services {
> >      namespace init {
> > -
> > +      using stan::interface_callbacks::writer::base_writer;
> There's no typedef. Just a using.
> 
> —
> Reply to this email directly or view it on GitHub.
>